src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/ConnectionTestParametersData.cs

```diff
@@ -0,0 +1,336 @@
+        {
+            yield return new object[] { Data.ConnectionTestParametersList[0] };
+            yield return new object[] { Data.ConnectionTestParametersList[1] };
+            yield return new object[] { Data.ConnectionTestParametersList[2] };
+            yield return new object[] { Data.ConnectionTestParametersList[3] };
+            yield return new object[] { Data.ConnectionTestParametersList[4] };
+            yield return new object[] { Data.ConnectionTestParametersList[5] };
+            yield return new object[] { Data.ConnectionTestParametersList[6] };
+            yield return new object[] { Data.ConnectionTestParametersList[7] };
+            yield return new object[] { Data.ConnectionTestParametersList[8] };
+            yield return new object[] { Data.ConnectionTestParametersList[9] };
+            yield return new object[] { Data.ConnectionTestParametersList[10] };
+            yield return new object[] { Data.ConnectionTestParametersList[11] };
+            yield return new object[] { Data.ConnectionTestParametersList[12] };
+            yield return new object[] { Data.ConnectionTestParametersList[13] };
+            yield return new object[] { Data.ConnectionTestParametersList[14] };
+            yield return new object[] { Data.ConnectionTestParametersList[15] };
+            yield return new object[] { Data.ConnectionTestParametersList[16] };
+            yield return new object[] { Data.ConnectionTestParametersList[17] };
+            yield return new object[] { Data.ConnectionTestParametersList[18] };
+            yield return new object[] { Data.ConnectionTestParametersList[19] };
+            yield return new object[] { Data.ConnectionTestParametersList[20] };
+            yield return new object[] { Data.ConnectionTestParametersList[21] };
+            yield return new object[] { Data.ConnectionTestParametersList[22] };
+            yield return new object[] { Data.ConnectionTestParametersList[23] };
+            yield return new object[] { Data.ConnectionTestParametersList[24] };
+            yield return new object[] { Data.ConnectionTestParametersList[25] };
+            yield return new object[] { Data.ConnectionTestParametersList[26] };
+            yield return new object[] { Data.ConnectionTestParametersList[27] };
+            yield return new object[] { Data.ConnectionTestParametersList[28] };
+            yield return new object[] { Data.ConnectionTestParametersList[29] };
+        }
+
+        public ConnectionTestParametersData() 
+        {
+            ConnectionTestParametersList = new List<ConnectionTestParameters>
+            {
+                new ConnectionTestParameters
+                {
+                    TdsEncryptionType = TDSPreLoginTokenEncryptionType.Off,
+                    Encrypt = SqlConnectionEncryptOption.Optional,
+                    TrustServerCertificate = false,
+                    Certificate = null,
+                    HostNameInCertificate = null,
```

Comment:
can we use the default value which is an empty string? Why null is selected?

Also can you add a comment description of the table being used in below cases?
something like possible options are:
<and add the options int he comment here>